### DISCUSSION TITLE
maplibre-gl-native on windows?

### DISCUSSION THREAD

**Discussion by catherineMom** [maplibre/maplibre-native#259] [Q&A]
Hello,

I got the source code compiled and run on windows machine, I had to comment out most of the code in headless-backend class, because it is based on glx, but I'm going to replace with Windows' equivalent later. 

Now, I want to run maplibre-render-test and see how the library works. Do you have any sample manifest file for me to use in the test application?  

Thank you..

**Comment by wipfli** [comment]
Are you using Qt?
